### ISSUE TITLE
Error in bs3_dialog("save_current_diag", "Save currently opened file?", : could not find function "bs3_dialog"

### ISSUE THREAD

**Issue by j450h1** [swarm-lab/editR#30]
I have a README.md file in the root of my RStudio project. See below. Any idea what could be going wrong?

``` r
editR::editR(here::here("README.md"))
#> Loading required package: tools
#> Loading required package: shiny
#> Error in bs3_dialog("save_current_diag", "Save currently opened file?", : could not find function "bs3_dialog"
sessionInfo()
#> R version 3.6.3 (2020-02-29)
#> Platform: x86_64-apple-darwin15.6.0 (64-bit)
#> Running under: macOS Catalina 10.15.4
#> 
#> Matrix products: default
#> BLAS:   /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRblas.0.dylib
#> LAPACK: /Library/Frameworks/R.framework/Versions/3.6/Resources/lib/libRlapack.dylib
#> 
#> locale:
#> [1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8
#> 
#> attached base packages:
#> [1] tools     stats     graphics  grDevices utils     datasets  methods  
#> [8] base     
#> 
#> other attached packages:
#> [1] rmarkdown_2.1    knitr_1.28       shinyBS_0.61     shinyFiles_0.8.0
#> [5] shinyAce_0.4.1   shiny_1.4.0.2   
#> 
#> loaded via a namespace (and not attached):
#>  [1] Rcpp_1.0.4.6     magrittr_1.5     xtable_1.8-4     here_0.1-11     
#>  [5] R6_2.4.1         rlang_0.4.6      fastmap_1.0.1    stringr_1.4.0   
#>  [9] highr_0.8        xfun_0.13        ellipsis_0.3.1   htmltools_0.4.0 
#> [13] yaml_2.2.1       digest_0.6.25    rprojroot_1.3-2  lifecycle_0.2.0 
#> [17] tibble_3.0.1     crayon_1.3.4     later_1.0.0      vctrs_0.3.0.9000
#> [21] fs_1.4.1         promises_1.1.0   evaluate_0.14    mime_0.9        
#> [25] stringi_1.4.6    pillar_1.4.4     compiler_3.6.3   editR_0.2.2     
#> [29] backports_1.1.7  jsonlite_1.6.1   httpuv_1.5.2     pkgconfig_2.0.3
```

<sup>Created on 2020-05-16 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by sjmgarnier** [comment]
@j450h1 Thanks for reporting this problem. I have completely stopped the development of editR more than 4 years ago (not enough time + R Notebooks in RStudio pretty much solve the problem I wanted to address with editR) and nobody picked the project up after I discontinued it. Many packages that editR depends on have probably been updated since I stopped working on it and there are probably many incompatibilities that have accumulated since then. I'm afraid I can't help more than that.
